pom.xml

```diff
@@ -199,6 +199,13 @@
             <version>1.0.0</version>
         </dependency>
 
+        <!-- Other -->
+        <dependency>
+            <groupId>za.co.absa.commons</groupId>
+            <artifactId>commons_${scala.compat.version}</artifactId>
+            <version>0.1.0</version>
+        </dependency>
+
```

Comment:
Is this some kind of merging error? The same dependency is declared immediately before.

Reply:
Yes you're right, this shouldn't be in here and will be taking it out.

Reply:
Removed duplicate entry